packages/p2p/src/components/verification/verification.tsx

```diff
@@ -23,10 +21,10 @@ const VerificationWrapper = ({ should_wrap, children }) => {
         );
     }
 
-    return children;
+    return children as JSX.Element;
```

Comment:
suggestion: we should avoid using `as` to assert type if possible, maybe declare the `children` type within the prop type `TVerificationWrapperProps`?

Reply:
<img width="931" alt="Screenshot 2023-05-30 at 10 53 40 AM" src="https://github.com/farrah-deriv/deriv-app/assets/122768621/79f18cf8-401e-45d5-9197-4d933f3df694">

it gives this error

Reply:
have you tried wrapping the last return with an empty fragment or React.Fragment? maybe this could be causing the issue